examples/squeezer_bin/squeezer_bin/imp.rs

```diff
@@ -0,0 +1,111 @@
+use gtk::glib;
+use gtk::gsk;
+use gtk::prelude::*;
+use gtk::subclass::prelude::*;
+
+use once_cell::sync::Lazy;
+use std::cell::Cell;
+
+#[derive(Debug)]
+pub struct SqueezerBin {
+    pub keep_aspect_ratio: Cell<bool>,
+}
+
+#[glib::object_subclass]
+impl ObjectSubclass for SqueezerBin {
+    const NAME: &'static str = "SqueezerBin";
+    type ParentType = gtk::Widget;
+    type Type = super::SqueezerBin;
+
+    fn new() -> Self {
+        Self {
+            keep_aspect_ratio: Cell::new(false),
+        }
+    }
+}
+
+impl ObjectImpl for SqueezerBin {
+    fn properties() -> &'static [glib::ParamSpec] {
+        static PROPERTIES: Lazy<Vec<glib::ParamSpec>> = Lazy::new(|| {
+            vec![glib::ParamSpecBoolean::new(
+                "keep-aspect-ratio",
+                "Keep aspect ratio",
+                "If true it keeps the aspect ratio of the widget",
+                false,
+                glib::ParamFlags::READWRITE | glib::ParamFlags::EXPLICIT_NOTIFY,
+            )]
+        });
+
+        PROPERTIES.as_ref()
+    }
+
+    fn property(&self, _obj: &Self::Type, _id: usize, pspec: &glib::ParamSpec) -> glib::Value {
+        match pspec.name() {
+            "keep-aspect-ratio" => self.keep_aspect_ratio.get().to_value(),
+            _ => unimplemented!(),
+        }
+    }
+
+    fn set_property(
+        &self,
+        obj: &Self::Type,
+        _id: usize,
+        value: &glib::Value,
+        pspec: &glib::ParamSpec,
+    ) {
```

Comment:
In general, I tend to prefer including `glib::{Value, ParamSpec};` in scope to make this looks less aweful